### ISSUE TITLE
wiki domain seed not random enough

### ISSUE THREAD

**Issue by toban** [wbstack/api#317]
From time to time CI fails because of domains seems to be getting re-used

```
1) Tests\Jobs\DeleteWikiDispatcherJobTest::testNothingDispatchesUntilItsTime
Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'harvey.com' for key 'wikis_domain_unique' (SQL: insert into `wikis` (`sitename`, `domain`, `deleted_at`, `updated_at`, `created_at`) values (Non., harvey.com, ?, 2022-01-12 08:44:04, 2022-01-12 08:44:04))

/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:705
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:665
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:495
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:447
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Query/Processors/Processor.php:32
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2966
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:1655
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1188
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1153
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:994
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:314
/var/www/html/vendor/laravel/framework/src/Illuminate/Collections/Traits/EnumeratesValues.php:242
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:317
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:261
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:255
/var/www/html/tests/Jobs/DeleteWikiDispatcherJobTest.php:43
/var/www/html/tests/Jobs/DeleteWikiDispatcherJobTest.php:80
phpvfscomposer:///var/www/html/vendor/phpunit/phpunit/phpunit:97

Caused by
PDOException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'harvey.com' for key 'wikis_domain_unique'

/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:494
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:698
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:665
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:495
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:447
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Query/Processors/Processor.php:32
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2966
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:1655
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1188
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1153
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:994
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:314
/var/www/html/vendor/laravel/framework/src/Illuminate/Collections/Traits/EnumeratesValues.php:242
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:317
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:261
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:255
/var/www/html/tests/Jobs/DeleteWikiDispatcherJobTest.php:43
/var/www/html/tests/Jobs/DeleteWikiDispatcherJobTest.php:80
phpvfscomposer:///var/www/html/vendor/phpunit/phpunit/phpunit:97

ERRORS!
Tests: 89, Assertions: 218, Errors: 1, Skipped: 3.
```

**Comment by toban** [comment]
Should be solved by #322

**Comment by toban** [comment]
This just happened again, seems it's not random enough still

```

1) Tests\Jobs\DeleteWikiJobTest::testFailure with data set #1 (1, null, 'Wiki <WIKI_ID> is not marked ...etion.')
Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'voluptatem_.wiki.opencura.com' for key 'wikis_domain_unique' (SQL: insert into `wikis` (`sitename`, `domain`, `deleted_at`, `updated_at`, `created_at`) values (Eusebio Weissnat, voluptatem_.wiki.opencura.com, ?, 2022-01-18 07:15:45, 2022-01-18 07:15:45))

/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:712
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:672
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:502
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:454
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Query/Processors/Processor.php:32
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2999
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:1655
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1188
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1153
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:994
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:314
/var/www/html/vendor/laravel/framework/src/Illuminate/Collections/Traits/EnumeratesValues.php:242
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:317
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:261
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:255
/var/www/html/tests/Jobs/DeleteWikiJobTest.php:157
phpvfscomposer:///var/www/html/vendor/phpunit/phpunit/phpunit:97

Caused by
PDOException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'voluptatem_.wiki.opencura.com' for key 'wikis_domain_unique'

/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:501
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:705
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:672
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:502
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:454
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Query/Processors/Processor.php:32
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2999
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:1655
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1188
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1153
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:994
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:314
/var/www/html/vendor/laravel/framework/src/Illuminate/Collections/Traits/EnumeratesValues.php:242
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:317
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:261
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:255
/var/www/html/tests/Jobs/DeleteWikiJobTest.php:157
phpvfscomposer:///var/www/html/vendor/phpunit/phpunit/phpunit:97

ERRORS!
Tests: 89, Assertions: 223, Errors: 1, Skipped: 3.
Error: Process completed with exit code 2.
```
